src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1941,6 +1962,56 @@ private List<T> FilterModuleCollection<T>(IEnumerable<T> moduleCollection)
             return filteredModuleCollection;
         }
 
+        private void PrepareNoClobberWinCompatModuleImport(string moduleName, ModuleSpecification moduleSpec, ref ImportModuleOptions importModuleOptions)
```

Comment:
This method is called `Prepare...` but it actually does the import, no? If so, I think it should just be called something like `ImportWinCompatModule` or `ImportWinCompatModuleNoClobber`

Reply:
Well... it **conditionally** *imports* a core version of the module **as preparation** for *WinCompat import* of the WinPS module. :)
I don't know what is the best name for this...